cypress/support/lpa-commands.js

```diff
@@ -23,3 +23,32 @@ Cypress.Commands.add("createInvestigation", (lpaId) =>
 Cypress.Commands.add("putInvestigationOnHold", (investigationId) =>
   cy.postToApi(`/api/v1/investigations/${investigationId}/hold-periods`, {reason: "Police Investigation"}).its("body")
 );
+
+const waitForStableDOM = (iteration = 0) => {
+  let mutation,
+    pollInterval = 1000,
+    timeout = 60000;
+
+  const observer = new MutationObserver((m) => mutation = m);
```

Comment:
Dimitar, what is a MutationObserver - exactly what it says in terms of waiting for something to change.